packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -83,6 +82,7 @@ export const App = ({ flowQueryParams, navigatorLanguages }: AppProps) => {
               to="/settings#connected-services"
               noThrow
             />
+            <Redirect from="/avatar/change" to="/settings/avatar/" noThrow />
```

Comment:
So does `Redirect`'s `from` follow the [`HomePath`](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/constants/index.tsx#L6) and `to` does not? 🧐

Reply:
Yep, I believe because `Redirect` does not have to be a child of a Router, the `HomePath` does not get automatically prepended to `to`.